.github/CODEOWNERS

```
@@ -0,0 +1,71 @@
+# Centrifudge Code Owners
+
+# Changes to .github
+.github/* @mikiquantum @mustermeiszer @branan @NunoAlexandre @offerijns
```

I don't know much about how the workflows are setup.
```suggestion
.github/* @mikiquantum @mustermeiszer @branan @NunoAlexandre
```